### ISSUE TITLE
Calling `microphone.sound_level` once keeps the microphone LED on

### ISSUE THREAD

**Issue by microbit-carlos** [microbit-foundation/micropython-microbit-v2#175]
Described in: https://github.com/microbit-foundation/micropython-microbit-v2/issues/175#issuecomment-2017797401

**Comment by dpgeorge** [comment]
When calling `microphone.sound_level()`, the code should not register a listener on the CODAL message bus.  That's not required and is keeping the microphone LED on.

**Comment by microbit-carlos** [comment]
Yes, the `CODAL_STREAM_IDLE_TIMEOUT_MS` was implemented in https://github.com/lancaster-university/codal-core/commit/a1e1db692909a260ab341c89b292e085daecd259 and it's currently set to 75 ms: [lancaster-university/codal-core/inc/core/CodalConfig.h#L224](https://github.com/lancaster-university/codal-core/blob/992c0b11a0eb2a1edca9c2f76821f89a99a3acec/inc/core/CodalConfig.h#L224)
